src/zenml/models/hub_plugin_models.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Models representing ZenML Hub plugins."""
+
+from datetime import datetime
+from typing import TYPE_CHECKING, List, Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from zenml.utils.enum_utils import StrEnum
+
+if TYPE_CHECKING:
+    pass
+
+
+class PluginStatus(StrEnum):
+    """Enum that represents the status of a plugin."""
+
+    PENDING = "pending"  # Plugin is being built
```

Comment:
I think those comments might be nice in the docstring so they should up in the API docs?

Reply:
makes sense, done